packages/ws-worker/src/util/convert-lightning-plan.ts

```diff
@@ -53,15 +54,20 @@ export default (
 
   // But some need to get passed down into the engine's options
   const engineOpts: WorkerRunOptions = {};
-
   if (run.options) {
-    if (run.options.run_timeout_ms) {
+    if ('run_timeout_ms' in run.options) {
       engineOpts.runTimeoutMs = run.options.run_timeout_ms;
     }
-    if (run.options.sanitize) {
+    if ('payload_memory_limit_mb' in run.options) {
+      engineOpts.payloadLimitMb = run.options.payload_memory_limit_mb;
+    }
```

Comment:
What's the naming thought here? It's not really a memory limit right? What about `dataclip_limit_mb` or something like that?

Reply:
Good catch, there are too many words here! I'll fix it shortly 

I'm preferring payload to dataclip because it affects logs too